db/migrations/20220808173704-create-foreign-key-for-comment.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+    async up(queryInterface, Sequelize) {
+        const isCommentTableExists = await queryInterface.tableExists('comment');
+        const isCityTableExists = await queryInterface.tableExists('city');
+        const isCityIdAlreadyExists = await queryInterface.describeTable('comment');
+        if (isCommentTableExists && isCityTableExists && !isCityIdAlreadyExists.cityId) {
+            await queryInterface.addColumn('comment', 'cityId', {
+                type: Sequelize.INTEGER,
+                onDelete: 'CASCADE',
+                allowNull: false,
+                references: { model: 'city', key: 'id' },
+            });
+        }
+    },
+
+    async down(queryInterface, Sequelize) {
+        const isCityIdExists = await queryInterface.describeTable('comment');
+        if (isCityIdExists.cityId) {
```

Comment:
if isCityIdExists = null
null.cityId = error

Fix it

Reply:
and u get 'comment' but variable is city

Reply:
and you can have this tabel but this table can not have column 
you shoud add one more check

Reply:
> and u get 'comment' but variable is city

Yeah, but i getting comment for city id in it